source/libs/sync/src/syncMain.c

```diff
@@ -2733,7 +2736,7 @@ int32_t syncNodeOnClientRequestCb(SSyncNode* ths, SyncClientRequest* pMsg, SyncI
     code = ths->pLogStore->syncLogAppendEntry(ths->pLogStore, pEntry);
     if (code != 0) {
       // del resp mgr, call FpCommitCb
-      ASSERT(0);
+      sError("vgId:%d, failed to append log entry since %s",  ths->vgId, tstrerror(terrno));
```

Comment:
Asserts replaced with error codes and relevant log messages in a lot of similar places, esp. for error handling conditions.  However, it may be all right to place an assertion on some internal invariant of logic, to detect the errors of logic assumptions alarmingly, which the correctness of code is relied upon.